webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,658 @@
+            </Box>
+          </MenuItem>
+        ))}
+      </Select>
+    </Box>
+  );
+
+  return errorFetchingBasePipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingBasePipelineData.message}
+    </Typography>
+  ) : errorFetchingComparedPipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingComparedPipelineData.message}
+    </Typography>
+  ) : (
+    <Box
+      border={(theme) => `1px solid ${theme.palette.grey[200]}`}
+      height="100%"
+      padding={4}
+      marginTop={2}
+    >
+      <Box marginBottom={1} display="flex" justifyContent="space-between">
+        <Box width={340}>
+          <DatasetSplitToggler
+            availableDatasetSplits={availableDatasetSplits}
+            value={selectedDatasetSplit}
+            onChange={setSelectedDatasetSplit}
+          />
+        </Box>
+        {config && (
+          <Box display="flex" flexDirection="row" alignItems="center">
+            <FormControlLabel
+              label={`Compare Baseline (${
+                config?.pipelines?.[pipeline.pipelineIndex].name
+              } ) with:`}
+              labelPlacement="start"
+              sx={{ gap: 1, paddingRight: 2 }}
+              control={
+                <PipelineSelect
+                  selectedPipeline={comparedPipeline ?? ""}
+                  onChange={(value) => setComparedPipeline(value)}
+                  pipelineMenuItem={config.pipelines?.map(
+                    ({ name }, pipelineIndex) =>
+                      pipelineIndex !== pipeline.pipelineIndex && (
+                        <MenuItem key={pipelineIndex} value={pipelineIndex}>
+                          {name}
+                        </MenuItem>
+                      )
+                  )}
+                />
+              }
+            ></FormControlLabel>
+          </Box>
+        )}
+      </Box>
+      <Box
+        border={(theme) => `1px solid ${theme.palette.grey[200]}`}
+        sx={{ backgroundColor: (theme) => theme.palette.background.paper }}
+      >
+        <Table
+          sx={{
+            "& .MuiDataGrid-cell": {
+              borderBottom: "none",
+            },
+            "& .MuiDataGrid-columnHeaders": {
+              borderBottom: "none",
+            },
+            "& .MuiDataGrid-virtualScroller": {
+              // Stops accidental navigation on horizontal scroll with touch pad
+              overscrollBehaviorX: "contain",
+            },
+            "& .overall": {
+              borderRight: (theme) => `1px solid ${theme.palette.grey[300]}`,
+              background: (theme) => theme.palette.grey[200],
+            },
+            "& .delta": {
+              background: (theme) => theme.palette.grey[100],
+            },
+            "& .content-sticky": {
```

Comment:
What do you mean by the word "content"? I am wondering if it shouldn't just be `sticky`:
```suggestion
            "& .sticky": {
```

Reply:
Content. I meant to write the "page content" for code readability.